web/components/canvas/selectable-layer.tsx

```diff
@@ -1,60 +1,53 @@
 import React, { useState } from 'react';
 import { Group, Rect } from 'react-konva';
 
-export function SelectableLayer(props: {
+interface ISelectableLayer {
   children: JSX.Element;
   width: number;
   height: number;
   selected?: boolean;
   onFocusChange?: (id: string, focus: boolean) => void;
-}) {
+}
+
+export const SelectableLayer: React.FC<ISelectableLayer> = ({
```

Comment:
What is the reason changing functions as to arrow functions?

Reply:
This is more intuitive - I wanted to reduce code duplications by passing size params once with `size`, an object with `width` & `height`. we can structurize it

Reply:
Well second thoughts on this one. I'll put this one too on the agenda. Merging 😎